aidants_connect_web/views/FC_as_FS.py

```diff
@@ -82,13 +81,23 @@ def fc_callback(request):
     }
 
     headers = {"Accept": "application/json"}
+
     request_for_token = python_request.post(token_url, data=payload, headers=headers)
     content = request_for_token.json()
 
     fc_access_token = content.get("access_token")
 
     fc_id_token = content.get("id_token")
-    # TODO Check that ID token is good
+    fc_secret = settings.FC_AS_FS_SECRET
```

Comment:
Could this be directly in the `jwt.encode` arguments ?
(same question for `fc_id `)